presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStatsSummarizer.java

```diff
@@ -118,6 +126,12 @@ private PlanNodeStatsSummarizer() {}
                             (map1, map2) -> mergeMaps(map1, map2, OperatorHashCollisionsStats::merge));
                 }
 
+                // The only statistics we have for Window Functions are very low level, thus displayed only in VERBOSE mode
+                if (operatorStats.getInfo() instanceof WindowOperator.WindowInfo && verbose) {
```

Comment:
Merging is not really going to be very CPU intensive, and the data has already been transferred. We should make the change to collect everything and display according to display-specific settings.